### PR TITLE
Picture-In-Picture support, update and fixes

### PR DESCRIPTION
**Pull request by Maxou44**
I updated information about picture in picture mode in browsers and added resources for Opera, Android and Safari.

### CODE REVIEW

File: features-json/picture-in-picture.json [hunk 1]
```diff
@@ -175,7 +183,7 @@
       "66":"n",
       "67":"n",
       "68":"n",
-      "69":"n",
+      "69":"d #2",
```
Comment (Fyrd) [inline features-json/picture-in-picture.json]:
```suggestion
      "69":"n d #2",
```
Need to include the "n" value

File: features-json/picture-in-picture.json [hunk 2]
```diff
@@ -329,16 +337,19 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture."
+    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture.",
+	"2":"Enabled in Chrome through the \"#enable-picture-in-picture\" and \"#enable-surfaces-for-videos\" flag in chrome://flags",
```
Comment (beaufortfrancois) [inline features-json/picture-in-picture.json]: I'm not sure what is the value of this note. There were some Picture-in-Picture bugs in Chrome 69. These flags were temporary.

Comment (Maxou44) [reply features-json/picture-in-picture.json]: It's just an additionnal note for Chrome 69. This kind of information are removed when the feature is published in the browser without flag?

Comment (beaufortfrancois) [reply features-json/picture-in-picture.json]: Picture-in-Picture is now working without any flag in Chrome 70. Flags will be removed soon.

Comment (Maxou44) [reply features-json/picture-in-picture.json]: So we remove the information for Chrome 69?

Comment (beaufortfrancois) [reply features-json/picture-in-picture.json]: I think so yes.

File: features-json/picture-in-picture.json [hunk 3]
```diff
@@ -337,8 +337,7 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture.",
-	"2":"Enabled in Chrome through the \"#enable-picture-in-picture\" and \"#enable-surfaces-for-videos\" flag in chrome://flags",
+    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture."
```
Comment (HM100) [inline features-json/picture-in-picture.json]: You missed a comma

Comment (beaufortfrancois) [reply features-json/picture-in-picture.json]: @Maxou44 (gentle ping)

Comment (HM100) [reply features-json/picture-in-picture.json]:
```suggestion
    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture.",
```

Comment (Maxou44) [reply features-json/picture-in-picture.json]: Thanks for the fix 👌

Comment (Fyrd) [reply features-json/picture-in-picture.json]: @Maxou44 I think you need to actually accept/make the suggested fix, I'm not seeing it as a PR commit.

**Comment by Fyrd** [comment]
Not gonna merge while there's still failures. Also can we get note 3 & 4 to be 2 & 3? Bit confusing otherwise.

**Comment by beaufortfrancois** [comment]
@Maxou44 Can you fix failures? If not, I'm happy to send a PR.

**Pull request merged**